### ISSUE TITLE
Re-export and document `BitFlags` usage.

### ISSUE THREAD

**Issue by Levitanus** [helgoboss/reaper-rs#70]
I found in tests.rs, that I cannot simply use `BitFlags` struct. I think, that I should include them as dependency in `Cargo.toml`, than try to construct in the test function.

I think — it is too complicated and not very clear for the end-user. Maybe consider re-export of `BitFlags` and writing a couple of strings in docs?

**Comment by helgoboss** [comment]
Yes, I think that's a good idea. At the moment one needs to include it manually making sure it's the same version.
